From cb0fd0567c30a101b9bb132822854ec6bd274295 Mon Sep 17 00:00:00 2001
From: stelukun <sreedhar.telukuntla@intel.com>
Date: Mon, 3 Jun 2019 20:25:30 +0530
Subject: [PATCH 1372/1494] drm/i915: Supress Waddress-of-packed-member
 warnings

Unable to get the i915 debug traces With option -Werror
enabled, as the kernel build is failing. This change helps
in skipping the warnings associated with packed member
address access and let the build to complete.

Signed-off-by: stelukun <sreedhar.telukuntla@intel.com>
---
 drivers/gpu/drm/i915/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/Makefile b/drivers/gpu/drm/i915/Makefile
index c70a494..9e145c9 100644
--- a/drivers/gpu/drm/i915/Makefile
+++ b/drivers/gpu/drm/i915/Makefile
@@ -22,7 +22,7 @@ subdir-ccflags-y += $(call cc-disable-warning, unused-but-set-variable)
 subdir-ccflags-y += $(call cc-disable-warning, sign-compare)
 subdir-ccflags-y += $(call cc-disable-warning, sometimes-uninitialized)
 subdir-ccflags-y += $(call cc-disable-warning, initializer-overrides)
-subdir-ccflags-$(CONFIG_DRM_I915_WERROR) += -Werror
+subdir-ccflags-$(CONFIG_DRM_I915_WERROR) += -Werror -Wno-address-of-packed-member
 
 # Fine grained warnings disable
 CFLAGS_i915_pci.o = $(call cc-disable-warning, override-init)
-- 
2.7.4

